packages/ozone/src/api/team/addMember.ts

```diff
@@ -26,14 +26,23 @@ export default function (server: Server, ctx: AppContext) {
           )
         }
 
+        const appviewCreator = {
+          agent: ctx.appviewAgent,
+          createAuthHeaders: ctx.appviewAuth,
+        }
+        const profiles = await teamService.getProfiles([did], appviewCreator)
```

Comment:
It would be kind of nice if adding a member would still complete even if fresh profile information couldn't be fetched.

Reply:
yeah `teamService.getProfiles` handles failures internally and won't throw error if fetching profile fails.